stdlib/2/pydoc.pyi

```diff
@@ -15,11 +16,11 @@ __version__ = ...  # type: str
 __credits__ = ...  # type: str
 
 def pathdirs() -> List[str]: ...
-def getdoc(object: object) -> str: ...
-def splitdoc(doc) -> Tuple[str, str]: ...
+def getdoc(object: object) -> AnyStr: ...
```

Comment:
Right, then it should be `Union[str, unicode]`. I think, anyway -- the code uses `inspect.getdoc` and `inspect.getcomments`, which in typeshed both have `str` as a return type. But I'm not sure those are correct, since you can have unicode doc comments.